imap_processing/hi/hi_cdf_attrs.py

```diff
@@ -50,87 +33,102 @@
     validmin=0,
     validmax=10,
     format="I2",
-    label_axis="ESA step",
+    label_axis="Energy Step",
     display_type="time_series",
     catdesc=(
-        "ESA step. "
+        "ESA step (0-10), nominally 9, but possibly 8 or 10,"
+        " 0 is likely a background test"
+    ),
+    var_notes=(
+        "ESA (electrostatic analyzer) step. "
         "It's a 4-bits integer value that represents the "
         "ESA step. ESA step value range from 0-10. "
         "nominally 9 ESA steps, but possibly 8 or 10. "
-        "'Step 0' is likely to refer to a background test."
+        "'Step 0' is likely to refer to a background test. "
+        "This value is used to look up the actual energy "
+        "value from its lookup table."
     ),
-    fieldname="ESA step",
+    fieldname="ESA step number (0-10)",
     fill_val=GlobalConstants.INT_FILLVAL,
-    var_type="metadata",
+    var_type="support_data",
     depend_0="epoch",
 )
 
 de_tag_attrs = ScienceAttrs(
-    validmin=GlobalConstants.INT_FILLVAL,
-    validmax=GlobalConstants.INT_MAXVAL,
-    format="I6",
-    label_axis="Direct event time tag",
+    validmin=0,
+    validmax=GlobalConstants.UBIT16_MAXVAL,
+    format="I5",
+    label_axis="Time Tag",
     display_type="time_series",
-    catdesc=(
+    catdesc=("Direct event time tag value"),
+    var_notes=(
         "Direct event tag. "
         "It's a 16-bits integer value that represents the "
         "direct event time tag."
     ),
-    fieldname="Direct event tag",
+    fieldname="Direct Event Time Tag",
     fill_val=GlobalConstants.INT_FILLVAL,
-    var_type="metadata",
+    var_type="support_data",
     depend_0="epoch",
 )
 
 trigger_id_attrs = ScienceAttrs(
-    validmin=GlobalConstants.INT_FILLVAL,
+    validmin=1,
     validmax=3,
     format="I1",
-    label_axis="Trigger ID",
+    label_axis="ID",
     display_type="time_series",
     catdesc=(
+        "Trigger ID of which detector was hit first. 1,2,3 for detector A,B,C resp"
```

Comment:
Just to verify they count 1-3 and not 0-2?
```suggestion
        "Trigger ID of which detector was hit first. 1, 2, or 3 for detector A, B, or C respectively"
```

Reply:
I had to remove those spaces and shorten respectively to fit the 80 char limit feedback from Andriy but also have meaningful comment.

Reply:
Ahh, interesting! Sorry I didn't realize this field was character limited. Would it be OK to just put "Trigger ID of the detector that was hit first" and leave the 1/2/3 text to the following longer text section?